spec/lib/miq_automation_engine/methods/wait_for_ip_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+require Rails.root.join('db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/wait_for_ip').to_s
+require Rails.root.join('spec/support/miq_ae_mock_service').to_s
```

Comment:
@Fryguy 
The problem was I had an exit in the Method and since I am testing this outside of the Automate Engine the exit from the Method was bailing the rspec.